app/services/testdata/faker/DataFaker.scala

```diff
@@ -772,6 +772,8 @@ object DataFaker {
           skillSelected -> numberOfPeopleWithSkillsRequired
         }.toMap
       }
+
+      def sessions = randList(List(Session("First", startTime, startTime.plusHours(1))),1)
```

Comment:
Not sure if it is necessary at this moment but you can combine randList with number like this:

def sessions = randList(List(Session("First", startTime, startTime.plusHours(1))),number(2))

where 2 means to number of maximum of sessions per day.

There is a method called number(Int) in DataFaker.Random object that generates a random integer.